packages/libs/error-stack/src/ext/result.rs

```diff
@@ -46,124 +46,50 @@ pub trait ResultExt {
     /// Type of the [`Ok`] value in the [`Result`]
     type Ok;
 
-    /// Adds new contextual information to the [`Frame`] stack of a [`Report`].
+    /// Adds a new attachment to the [`Result`].
```

Comment:
Do we want to be more specific with these

```suggestion
    /// Adds a new attachment to [`Report`] inside the [`Result`].
```